### PR TITLE
Bring travis test up to date

### PR DESCRIPTION
**Pull request by bwrsandman**
Flake and coveralls are installed and run automatically

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,12 +26,9 @@ install:
   - git clone --depth=1 https://github.com/OCA/account-invoicing.git ${HOME}/account-invoicing -b ${VERSION}
   - git clone --depth=1 https://github.com/OCA/bank-statement-reconcile.git ${HOME}/bank-statement-reconcile -b ${VERSION}
   - git clone --depth=1 https://github.com/OCA/webkit-tools.git ${HOME}/webkit-tools -b ${VERSION}
-  
 
 script:
-  - travis_run_flake8
```
Comment (max3903) [inline .travis.yml:32]: No more PEP8 tests ?

Comment (bwrsandman) [inline .travis.yml:32]: The tests are integrated within `travis_run_tests` (the next command) now.

**Comment by max3903** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**